distribution/docker/alpine/Dockerfile

```
@@ -3,12 +3,12 @@ FROM alpine:3.14 AS builder
 WORKDIR /vector
 
 COPY vector-*-unknown-linux-musl*.tar.gz ./
-RUN tar -xvf vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz --strip-components=2
+RUN tar -xvf "vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz --strip-components=2"
```

I don't think the quotes are correct here? Shouldn't it be:
```suggestion
RUN tar -xvf "vector-0*-$(cat /etc/apk/arch)-unknown-linux-musl*.tar.gz" --strip-components=2
```

---

Fixed f1c132ef8

---

See my last comment, these quotes are actually undesirable 🙈

---

_Feel like you're just trolling me now_ :P

---

My brain is trolling myself first and foremost!
Those quotes did smell weird at first sight, but then got distracted by their positioning! >_<

---

Yeah - they're _extra_ wrong regardless. I'll confirm your expectations and see that the quoting breaks things here.

---

I confirmed ìt (and remembered from my bash-fu) before commenting haha:
```
$ ls -d "*"
ls: cannot access '*': No such file or directory
```

---

Ah yes, that's an idiomatic fix indeed!
It protects the _variable expansion_ but leaves the _globbing_ intact. 💯

---

_Just need to keep making the quoted section smaller_ 🤌

---

Fixed for real ™️ e2c3e7e57